src/Pack/Runner.idr

```diff
@@ -179,3 +183,6 @@ runCmd = do
         env <- idrisEnv mc fetch
         install []
         writeCollection
+      (Uninstall ** [])         => do
+        env <- idrisEnv mc fetch
+        uninstallPack
```

Comment:
We don't need the Idris to be installed or collection to be resolved in order to remove everything

```suggestion
      (Uninstall ** [])         => uninstallPack
```

But currently this won't work because of idris-lang/Idris2#3365, we can workaround it with

```suggestion
      (Uninstall ** [])         => uninstallPack @{%search} @{metaConfigToLogRef @{mc}}
```

but we'd rather wait for the fixing PR to be merged rather than injecting ugly workarounds.

Reply:
Thanks for catching this, should be addressed via 1b99a87fb55a97b8ede6fbfb1a6e2d6d820e756d.

Reply:
Right. Since, there is another issue of the compiler (idris-lang/Idris2#3370), we cannot remove explicit part of the auto-implicit argument. What I suggest is to additionally swap two `auto`-implicit arguments to make the one we need to specify to be the first (I pushed it to your branch myself).

Reply:
I think, since it is an uneasy upstream problem, this code is okay, to let's leave it as is